src/System.IO.Compression/tests/System.IO.Compression.Tests.csproj

```diff
@@ -45,8 +49,11 @@
       <Link>Common\System\IO\Compression\ZipTestHelper.cs</Link>
     </Compile>
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == 'netcoreapp'">
```

Comment:
Disregard the above --this is a test configuration. I believe you did it right. My bad.